src/store/RelayQueryTracker.js

```diff
@@ -60,8 +60,16 @@ class RelayQueryTracker {
     const {isMerged, trackedNodes} = trackedNodesByID;
     if (!isMerged) {
       const trackedChildren = [];
+      let wrapperNode = null;
       trackedNodes.forEach(trackedQuery => {
-        trackedChildren.push(...trackedQuery.getChildren());
+        wrapperNode = new RelayQuery.Fragment.build(
+          'RelayQueryTracker',
+          trackedQuery.getType(),
+          trackedQuery.getChildren()
+        );
+        if (wrapperNode) {
```

Comment:
Ok seemed useless to me too. I was mimicking [this](https://github.com/mattecapu/relay/blob/fae833a8e422206f7936a4b4363c07546cfea66d/src/store/RelayQueryTracker.js#L87) snippet of code.